### PR TITLE
bpo-34124: fix message_from_binary_file typo

### PR DESCRIPTION
**Pull request by jribbens**
The documentation for `email.message_from_binary_file` is missing a line-continuation backslash at the end of the `.. function::` line which means the output formatting is mangled and it has no permalink.

https://bugs.python.org/issue34124

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately our records indicate you have not signed the CLA. For legal reasons we need you to sign this before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.


When your account is ready, please add a comment in this pull request
and a Python core developer will remove the `CLA not signed` label
to make the bot check again.

Thanks again for your contribution, we look forward to reviewing it!

**Review by berkerpeksag — APPROVED** [review]
Good catch, thanks! Did you signed the CLA? Since this is a pretty trivial patch, we don't need to wait for the PSF approval, so I only need to know whether you signed the CLA :)

**Comment by jribbens** [comment]
@berkerpeksag I signed the CLA shortly before the pull request, so I expect it's currently in the process of being looked at. I suspect however that the chances of my successfully claiming copyright on this two-byte patch is low ;-)

**Comment by berkerpeksag** [comment]
Thanks!

**Pull request merged**
